README.md

```diff
@@ -359,6 +359,7 @@ Nuxt.js is a framework for creating Universal Vue.js Applications.
 - [Beerbay](https://beerbay.net/) - Beerbay is a craft beer e-commerce built with NUXT and WP api.
 - [alibaba.aero](https://www.alibaba.aero) - Alibaba Travels Co is Iran's leading travel website built with Nuxt.js.
 - [vue-org-chart](https://github.com/Hoogkamer/vue-org-chart) - Manage and publish your interactive organization chart (orgchart), free and no webserver required.
+- [frontendjs](https://www.frontendjs.com/) - Front-end navigation website, including web sites, communities, documents about the front-end.
```

Comment:
I'd suggest to note that it's a chinese website as well.

Reply:
Websites that can only submit English content?